Cookbook/Technical-Documents/meetings-purpose.md

```diff
@@ -21,18 +21,18 @@ The schedule is the following:
 
 ## PR Parties
 
-The PR Parties are meetings where a group of engineers will get together to review a bunch of PRs they're assigned to. Usually the authors of these PRs will be in the group and they will go through their PRs and explain them.
-This will allow the authors to explain the solution, to open the discussion about it and to agree on an approach if required. In an ideal world, we would have all the PRs approved by the end of the meeting, but this is not the main goal. The purpose is to facilitate the discussion about the PR implementation.
+The PR Parties are groups of engineers that are assembled to review a bunch of PRs they're assigned to. Usually the authors of these PRs will be in the group. Participants will start by reviewing PRs asynchronously on their own. 
+If there is a need a meeting will be held where authors will go through their PRs and explain them. This will open the discussion about it and to agree on an approach if required. In an ideal world, we would have all the PRs approved by the end of the meeting, but this is not the main goal. The purpose is to facilitate the discussion about the PR implementation.

```

Comment:
```suggestion
If there is a need, a meeting will be organised where the authors will go through their PRs and explain them. This will open a discussion about them and a different approach can be agreed on if required. In an ideal world, we would have all the PRs approved by the end of the meeting, but this is not the main goal. The purpose is to facilitate the discussion about the PR implementation.
```